machida/lib/wallaroo/__init__.py

```diff
@@ -81,47 +81,54 @@ def from_source(class_object, name, source_config):
     def __to_tuple__(self, app_name):
         return self._pipeline_tree.to_tuple(app_name)
 
-    def __is_closed__(self):
+    def _is_closed(self):
         return self._pipeline_tree.is_closed
 
     def to(self, computation):
-        return self.clone().__to__(computation)
+        return self.clone()._to(computation)
 
-    def __to__(self, computation):
+    def _to(self, computation):
         if isinstance(computation, StateComputation):
             self._pipeline_tree.add_stage(("to_state", computation))
         else:
             self._pipeline_tree.add_stage(("to", computation))
         return self
 
     def to_sink(self, sink_config):
-        return self.clone().__to_sink__(sink_config)
+        return self.clone()._to_sink(sink_config)
 
-    def __to_sink__(self, sink_config):
+    def _to_sink(self, sink_config):
         self._pipeline_tree.add_stage(("to_sink", sink_config.to_tuple()))
         return self
 
     def to_sinks(self, sink_configs):
-        return self.clone().__to_sinks__(sink_configs)
+        return self.clone()._to_sinks(sink_configs)
 
-    def __to_sinks__(self, sink_configs):
+    def _to_sinks(self, sink_configs):
         sinks = []
         for sc in sink_configs:
             sinks.append(sc.to_tuple())
         self._pipeline_tree.add_stage(("to_sinks", sinks))
         return self
 
     def key_by(self, key_extractor):
-        return self.clone().__key_by__(key_extractor)
+        return self.clone()._key_by(key_extractor)
 
-    def __key_by__(self, key_extractor):
+    def _key_by(self, key_extractor):
         self._pipeline_tree.add_stage(("key_by", key_extractor))
         return self
 
+    def collect(self):
+        return self.clone()._collect()
+
+    def _collect(self):
+        self._pipeline_tree.add_stage(("collect", ""))
```

Comment:
Again, this is probably best defined as as CAPITALIZED_CONSTANT somewhere upstairs.

Reply:
Are you referring to `"collect"` here? You might be right, but since I'm following the pattern that already existed here, I think this one is better addressed in a separate issue.

Reply:
Gotcha. Yes, I was referring to the "collect' string there. Let's address it in a separate issue.